src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -57,11 +57,7 @@
 import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
 import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
 import org.glassfish.hk2.api.MultiException;
```

Comment:
We could also switch to Java8 and use Base64.getEncoder()

On Mon, 2 May 2016 at 18:19 Jeff Skjonsby <notifications@github.com> wrote:

> In src/main/java/com/spotify/docker/client/DefaultDockerClient.java
> <https://github.com/spotify/docker-client/pull/340#discussion_r61761412>:
>
> > @@ -57,11 +57,7 @@
> >  import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
> >  import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
> >  import org.glassfish.hk2.api.MultiException;
>
> MultiException is in hk2-api which doesn't seem to conflict with RESTEasy
> anyways so I don't think I need to worry about getting rid of the
> reference, although explicitly adding the maven dependency rather than
> using it transitively would be good. I'll make that change.
>
> Base64 I didn't notice and I assume would've failed if I tried to use it
> with registry authentication. Any objections to switching that to
> commons-codec?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/docker-client/pull/340/files/f540f50f336c198eca7ea006a9f59e5a5f0f0e30#r61761412>
>